src/styles/utils/variables.scss

```diff
@@ -0,0 +1,8 @@
+:root {
+  --card-button-color: #00acdc;
+  --card-border-color: #f3f3f3;
+  --text-color-button: #fff;
+  --text-color-secondary: #616070;
+  --text-color-main-accent: #060b35;
+  --color: #00acdc;
+}
```

Comment:
looks good, just one small last thing

you can use scss variables and no need to use `:root` then. use scss features